src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -12,19 +14,52 @@
 
 public class Operations {
 
-    public static <T extends IPerson> Set<T> getByFriendliness(Configuration<T> configuration, double value) {
+    /**
+     * Gives you a set of all players with that specific value.
+     * WARNING: This should not be used to get all friends or something like that. It's only there for completeness
+     *
+     * @param configuration the config.
+     * @param value that every player in the set should have.
+     * @param <T>
+     * @return all players that are saved as your enemy. No matter how hostile they are.
```

Comment:
Use links, document all parameters (you're missing a description for the generic type parameter `<T>`)

Try referring to the javadoc for `introduceNewSafe`:
```
* @param configuration The {@link Configuration} to mutate
* @param person        The {@link IPerson} to add
* @param <T>           The type of {@link IPerson} this configuration holds
* @return              The mutated {@link Configuration}
```